.github/workflows/main.yml

```diff
@@ -123,6 +133,10 @@ jobs:
           CF_FILE="ecsapi-demo-cloudformation.yaml"
           awslocal cloudformation create-stack --stack-name $STACK --template-body file://$CF_FILE
 
+      - name: Deploy Web Application
```

Comment:
NIT: the above step (cannot mark it correctly as it hasn't been changed) has the name "Deploy on Terraform", but it seems like a copy paste error, as it actually deploys using awslocal cloudformation? 🙂